resources/js/actions/userAuthActions.js

```diff
@@ -1,77 +1,240 @@
-import userApi from '../apis/userApi';
-import { SIGN_IN } from './types';
-
-export const signIn = (formData) => async dispatch => {
+import userApi from "../apis/userApi";
```

Comment:
Please use 2 spaces spacing settings for all `js` files. Follow video below how to do it in VSCode.



https://user-images.githubusercontent.com/21002044/154006895-3bf9f146-1ca6-485e-a66d-8c82b4848fe5.mov

Reply:
I have followed your instructions and implemented it ma'am. Thank you.

Reply:
@michaelsilverio still showing 4 spaces for all `js` files. You have to save each one to apply the changes.

Reply:
Sorry ma'am, I tried following your instructions and I thought it was fixed. It turns out that I have a problem with the Prettier and VS Code configuration because when I used the 2 space indentation it was not working. I had to tweak some settings and now it is finally fixed.